### PR TITLE
new(kernel_crawler): add support for AmazonLinux2023.

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind feature

**Any specific area of the project related to this PR?**

/area crawler

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: kernel_crawler/amazonlinux.py [hunk 1]
```diff
@@ -74,6 +74,7 @@ class AmazonLinux2022Mirror(repo.Distro):
     # This was obtained by running
     # docker run -it --rm amazonlinux:2022 python3 -c 'import dnf, json; db = dnf.dnf.Base(); print(json.dumps(db.conf.substitutions, indent=2))'
     AL2022_REPOS = [
+        'latest',
```
Comment (FedeDP) [inline kernel_crawler/amazonlinux.py:78]: We missed this :) It adds just one kernel in my tests.

**Comment by FedeDP** [comment]
This will need:
- [ ] companion driverkit PR: https://github.com/falcosecurity/driverkit/pull/268 + a new tag + bump on test-infra
- [ ] companion falco-driver-loader PR https://github.com/falcosecurity/falco/pull/2494
- [ ] companion test-infra PR https://github.com/falcosecurity/test-infra/pull/1099

File: kernel_crawler/amazonlinux.py [hunk 2]
```diff
@@ -49,6 +49,7 @@ class AmazonLinux2Mirror(repo.Distro):
     AL2_REPOS = [
         'core/2.0',
         'core/latest',
+        'extras/kernel-ng/latest',
```
Comment (FedeDP) [inline kernel_crawler/amazonlinux.py:52]: We missed this!

**Review by EXONER4TED — APPROVED** [review]
Good finds!

**Comment by FedeDP** [comment]
We are still investigating why, but here we need to manually add the approved label :P
/label approved

**Pull request merged**